perma_web/api2/urls.py

```diff
@@ -0,0 +1,67 @@
+from django.conf.urls import url
+from rest_framework.routers import APIRootView
+
+from perma.urls import guid_pattern
+
+import views
```

Comment:
I'm obviously not careful about this, but in theory I prefer the explicit relative import approach (`import .views`) for imports within a Django app.

We have three options here -- `import views`, `import api2.views`, or `import .views`.

`import views` is super vague -- it could refer to api/views.py or perma_web/views.py or some third party `views` library. You can imagine a scenario where this actually matters, like if you have `import celery` referring to a local celery.py file and then delete that file, and `import celery` still works but unexpectedly refers to something totally different.

`import api2.views` means you have to change all the import lines if you rename the app, and it doesn't immediately show the difference between within-app imports and cross-app imports:

    # one of these things is not like the other ...
    from perma.views import foo_helper
    from api2.views import foo

So my favorite is `import .views` -- it clearly expresses that this is a relative import within this modular, movable thing we call an app.

In practice it's such a big hairy deal to rename folders and files that this particular thing may never matter that much ...